logger/correlation.go

```diff
@@ -0,0 +1,68 @@
+package logger
+
+import (
+	atomic "github.com/ElrondNetwork/elrond-go/core/atomic"
+)
+
+var correlation logCorrelation
```

Comment:
logger just got to know about epochs/rounds and subrounds.

Reply:
This is true. A compromise. Less agnostic logger, a bit more tailored for our needs. In the future, we can re-design correlation elements as a generic map (without concepts such as epochs / rounds / sub-rounds), but let's see first if / how it improves our bug hunts.

Reply:
ok